app/helpers/jobs_helper.rb

```diff
@@ -89,15 +94,15 @@ def job_listed_on_usajobs?(job_id)
 
   def more_federal_jobs_title_and_url
     title = t :'searches.more_federal_job_openings'
-    url = 'https://www.usajobs.gov/JobSearch/Search/GetResults?PostingChannelID=USASearch'
+    url = 'https://www.usajobs.gov/Search/Results?hp=public&p=1'
     [title, url]
   end
 
   def url_for_more_agency_jobs(agency, job_id)
     case job_id
     when /^usajobs/
-      organization_codes = agency.joined_organization_codes
-      "https://www.usajobs.gov/JobSearch/Search/GetResults?organizationid=#{organization_codes}&PostingChannelID=USASearch&ApplicantEligibility=all"
+      organization_codes = agency.joined_organization_codes.gsub(/\;/,"&a=")
```

Comment:
The `Agency#joined_organization_codes` method takes a `separator` argument, so you can simplify this as:
```ruby
organization_codes = agency.joined_organization_codes('&a=')
```

Reply:
👍 good to know!